multimodal/src/autogluon/multimodal/matcher.py

```diff
@@ -283,7 +302,11 @@ def fit(
         Parameters
         ----------
         train_data
-            A dataframe containing training data.
+            A dataframe, containing the query data, response data, and their relevance. For example,
```

Comment:
does not support multiple query/document columns?

Reply:
query_data and response_data can be multi-columns.